src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyActionPlanServiceTest.kt

```diff
@@ -43,6 +43,7 @@ class NotifyActionPlanServiceTest {
 
   private fun notifyService(): NotifyActionPlanService {
     return NotifyActionPlanService(
+      "template",
```

Comment:
I think we should have a new test here for approved events, currently only assigned events are being tested

Reply:
good spot